src/hotspot/share/memory/arena.hpp

```diff
@@ -166,22 +167,6 @@ class Arena : public CHeapObj<mtNone> {
     }
   }
 
-  // Allocate with 'double' alignment. It is 8 bytes on sparc.
-  // In other cases Amalloc_D() should be the same as Amalloc_4().
-  void* Amalloc_D(size_t x, AllocFailType alloc_failmode = AllocFailStrategy::EXIT_OOM) {
```

Comment:
I'm happy to see this go.  That description is about as clear as mud; I have no idea what the intended semantics are for this name.  Oh, I see. Before SPARC removal (JDK-8244224), this had some seriously stale conditional code for 32bit SPARC.  I still don't understand what that was about, but don't really care either.  Yay code deletion.